README.md

```diff
@@ -73,40 +73,42 @@ Elastic search for Sylius.
             search_list:
                 filters:
                     - channel
-                    - taxon_slug
+                    - taxon_code
                     - price_range
                     - locale
-                    - attribute_values
                     - paginator
                     - search
+                    - attributes
                 repository: es.manager.default.product
         filters:
             channel:
                 type: choice
                 request_field: channel
                 document_field: channel_code
-            taxon_slug:
+            taxon_code:
```

Comment:
Shouldn't it be `taxonCode`? We can clean it later, but we should move to camelCase in all API responses.